packages/components/src/stories/SquiggleChart/Calculator.stories.tsx

```diff
@@ -11,10 +11,35 @@ type Story = StoryObj<typeof meta>;
 export const Basic: Story = {
   args: {
     code: `
-    f(a, b, c, d) = [a,b,c,d]
+    f(a, b, c, d) = {|f| normal(f*10+b, 1)}
```

Comment:
Can we keep the old version too? I liked that it was possible to check that calculators are properly reactive for all input types.

Reply:
(Changed. That said, I'd encourage you to feel free to adjust it as needed)